ImageSource/Local/LocalImageSource.swift

```diff
@@ -5,6 +5,7 @@ import MobileCoreServices
 public final class LocalImageSource: ImageSource {
     
     public let path: String
+    public let url: URL? = nil
```

Comment:
It's better to have just one source of truth. Let it be `url`, remove `path`.

In `init(path: ...` initialize `url` with `self.url = URL(fileURLWithPath: path)`. It will render if-else in `fullResolutionImageData` unneccessary.

Reply:
Это не работает в некоторых случаях когда передаешь `path`, он просто не находит файл. из-за этого нужно передавать именно `URL`.

Reply:
Я не говорю, что надо убирать init, который с URL'ом. Я про то, что незачем держать две переменных, в которых лежит одно и то же. Можно оставить для обоих случаев только url, а `public let path: String` превратить в `public var path: String { url.path }` для обратной совместимости.

Reply:
А что это за случаи, когда файл не находится? Это локальный файл?